src/Features/Core/Portable/PullMemberUp/QuickAction/InterfacePullerWithQuickAction.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.  
+
+using Microsoft.CodeAnalysis.Shared.Utilities;
+
+namespace Microsoft.CodeAnalysis.PullMemberUp.QuickAction
+{
+    internal class InterfacePullerWithQuickAction : AbstractMemberPullerWithQuickAction
+    {
+        protected override bool IsSelectedMemberDeclarationAlreadyInDestination(
+            INamedTypeSymbol destinationSymbol,
```

Comment:
can destination symbol be a different language from the starting language?  if so FindImplementationForInterfaceMember will lkely crash when passed a symbl from a different language.  it's likely we won't be able to support that scenario fully.  That's ok, as long as we don't crash.